api/api.go

```diff
@@ -394,6 +395,7 @@ func ProvisionerKey(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	w.Header().Set("Cache-Control", "private, no-store")
```

Comment:
This key is encrypted.

Reply:
I see that now. I'll remove the header here.